core/arch/arm/kernel/generic_boot.c

```diff
@@ -760,6 +777,9 @@ static void init_primary_helper(unsigned long pageable_part,
 	init_vfp_nsec();
 	if (init_teecore() != TEE_SUCCESS)
 		panic();
+#ifdef CFG_DT
+	dt_blob_addr = NULL;
```

Comment:
Proposal to add comment `/* dt no more reached, reset pointer to invalid */`. It is not obvious why this is cleared.
Even add a `reset_dt_references();` to remove `#ifdef CFG_xxx`.

Reply:
ok. I'll fix it.